packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -16,7 +16,7 @@ import * as Errors from "../src/common/errors";
 import { Months } from "../src/common/months";
 import { Classes as DateClasses, DateRange, DateRangePicker, IDateRangePickerProps } from "../src/index";
 
-describe("<DateRangePicker>", () => {
+xdescribe("<DateRangePicker>", () => {
```

Comment:
oh nice, i can't find those rules - is it possible to also lint for the `xit` and `xdescribe` variants?